Automattic/wp-calypso
# Segmentation Survey: Create query retrieve survey structure

Closes https://github.com/Automattic/wp-calypso/issues/89272

## Proposed Changes

* Create useSurveyStructureQuery and apply to the segmentation survey step

## Testing Instructions

* Should be tested together with D144618-code
* Apply this PR to your local
* Revert the last commit to access the testing route `git revert 258af6d`
* Go to http://calypso.localhost:3000/ecommerce-segmentation-survey?flags=ecommerce-segmentation-survey
* Verify if the GET segmentation-survey endpoint was called
* Verify if the retrieved questions are rendered as expected

## Pre-merge Checklist
